<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_second_window</class>
 <widget class="QDialog" name="Admin_second_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 230);
color : green;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Admin Panel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_user">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
color: black;
font-size : 13px;
font-weight : bold;</string>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_item">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127);
color: black;
font-size : 13px;
font-weight : bold;</string>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 17px;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight : bold;
font-size : 12px;
color : black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 17px;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_username">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight : bold;
font-size : 12px;
color : black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_email">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight : bold;
font-size : 12px;
color : black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 17px;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 17px;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_password">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>170</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight : bold;
font-size : 12px;
color : black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight : bold;
background-color: rgb(85, 170, 0);
font-size : 14px;
color : #fff;
border : none;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight : bold;
background-color: rgb(85, 170, 0);
font-size : 14px;
color : #fff;
border : none;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight : bold;
background-color: rgb(85, 170, 0);
font-size : 14px;
color : #fff;
border : none;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_user">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>621</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
